ansible/roles/stack-sunbird/templates/sl-assessments.env

```diff
@@ -0,0 +1,145 @@
+# Service Config
+HOST={{sl_assessments_host | default("localhost")}}
+PORT={{sl_assessments_port | default("7201")}} 
+LOG={{sl_assessments_log | default("debug")}}
+NODE_ENV={{sl_assessments_node_env}}
+ENABLE_DEBUG_LOGGING={{sl_assessments_enable_debug_logging | default("ON")}}
+ENABLE_BUNYAN_LOGGING={{sl_assessments_enable_bunyan_logging | default("OFF")}}
+APPLICATION_BASE_URL={{sl_assessments_application_base_url | default('/assessment/')}}
+MOBILE_APPLICATION_APP_TYPE={{sl_assessments_mobile_application_app_type | default("assessment")}}
+APPLICATION_BASE_HOST={{sl_assessments_application_base_host}}
+
+# Allow report request to run for longer time
+request_timeout_for_reports={{sl_assessments_request_timeout_for_reports | default("600000")}}
+SHIKSHALOKAM_USER_PROFILE_FETCH_ENDPOINT={{sl_assessments_shikshalokam_user_profile_fetch_endpoint | default("/api/user/v1/read")}}
+AUTHORIZATION={{sl_assessments_authorization}}
+MONGODB_URL={{sl_assessments_mongodb_url}}
+SHIKSHALOKAM_BASE_HOST={{sl_assessments_shikshalokam_base_host}}
+
+#Cloud Storage Configuration
+CLOUD_STORAGE={{sl_assessments_cloud_storage}}
+
+
+# Azure Cloud Configuration
```

Comment:
are all three are mandatory or optional?
Create a section called
```
##### Cloud config
# Note1
# Note2
var1
var2
```

Reply:
changed according to your comment